qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -760,17 +797,20 @@ def figure(
         self,
         figure_key: Union[str, int],
         file_name: Optional[str] = None,
+        image_only: Optional[bool] = True,
```

Comment:
I would like if we could remove the need for this kwarg so it always returns the new class

Reply:
Done.